### ISSUE TITLE
Mislabeled Visitor Object

### ISSUE THREAD

**Issue by erickgonzalez** [dotCMS/core#23823]
### Proposed Objective

Accessibility (WCAG)

### Proposed Priority

Priority 4 - Trivial

### dotCMS Version

22.03 LTS, 22.10.2, 23.01

### Problem Statement

When you print `$visitor` the `persona` object that is displayed is actually meant for the `possible persona` object, but if you print the object `$visitor.persona` only one is printed.

E.g:

```
Visitor{id=234884118, ipAddress=/201.203.6.90, selectedLanguage=en-us, locale=en_US, persona=[WinterEnthusiast, OceanEnthusiast, OceanEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, OceanEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, EcoEnthusiast, WinterEnthusiast, WinterEnthusiast, WinterEnthusiast, OceanEnthusiast, WinterEnthusiast, WinterEnthusiast, WinterEnthusiast, WinterEnthusiast], accruedTags=[snowmobile x 8, skiing x 8, snowboarding x 8, windsurfing x 4, diving x 4, surfing x 8, kite surfing x 8, french polynesia x 4, waterenthusiast:persona x 4, climbing x 18, hiking x 18, zip-lining x 18], userAgent=MAC_OS_X-CHROME10, device=COMPUTER, dmid=6f2155d5-6f22-4544-8a3d-619a843e7d1d, newVisitor=true, referrer=null, lastRequestDate=null, map={}}
```

### Steps to Reproduce

1- Create a Code Snippet widget with the following code:
```
$visitor
```
2- Add the widget to any page
3- Hit the page from the front end
4- Change to a diff persona

### Acceptance Criteria

We should change the label persona to `possiblePersonas` and also add the `persona`.

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

https://dotcms.zendesk.com/agent/tickets/109622
